.github/workflows/createNewVersion.yml

```diff
@@ -25,6 +26,30 @@ on:
         description: Webhook used to comment in slack
         required: true
 
+  workflow_dispatch:
+    inputs:
+      SEMVER_LEVEL:
+        description: One of {BUILD, PATCH, MINOR, MAJOR}
+        required: false
+        default: BUILD
+        type: string
+
+    outputs:
```

Comment:
Unless I'm mistaken, `outputs` is not a valid key for `workflow_dispatch` trigger. Same w/ `secrets`

Reply:
I think you are right. The GH actions check will most likely fail